handler_test.go

```diff
@@ -143,6 +149,20 @@ func TestHandlerValidSignature(t *testing.T) {
 	assert.Equal(t, 200, resp.Code)
 	assert.Contains(t, resp.Body.String(), "Hello, client")
 }
+func TestHandlerValidSignatureS3cmd(t *testing.T) {
+	h := newTestProxy(t)
+
+	req := httptest.NewRequest(http.MethodGet, "http://foobar.example.com", nil)
+	signRequest(req)
+	authorizationReq := req.Header.Get("Authorization");
+    authorizationReq =  strings.Replace(authorizationReq,", Signature",",Signature",1)
+    authorizationReq =  strings.Replace(authorizationReq,", SignedHeaders",",SignedHeaders",1)
+    req.Header.Set("Authorization",authorizationReq);
```

Comment:
please run `go fmt` or similar. The whitespace around the commas inside a string and as function argument is confusing to read + also fix indendation.